vignettes/manuscript.Rmd

```diff
@@ -264,24 +264,30 @@ To account for unique aspects of each lake and characterize lake productivity, w
 
+<!---are the predictors randomly chosen for each tree or each split.  I guess it makes more sense for it to be at each split but I hadn't been clear on this.--->
+
 Random forests are able to handle numerous correlated variables without a decrease in prediction accuracy; however, one possible shortcoming of this approach is that the resulting model may be difficult to interpret, thus selecting the most important variables is an important first step.  Several method have been proposed to do this with random forest.  For instance, this is a problem often faced in gene selection and in that field, a variable selection method based on random forest has been successfully applied and implemented in the R Language as the `varSelRF` package [@diaz2006gene], but this is limited to classification problems.  Additionally, others have suggested alternative variable importance measures, but this is only needed with a large number of categorical variables which are selected against with traditional random forest approach [@strobl2007bias].  
 
 In our case, we are predicting a continuous variable, chlorophyll *a*, directly thus `varSelRF`, does not apply, and all of our variables are continuous so the approach suggested by Strobl [-@strobl2007bias] is not necessary.  Thus we developed an approach, similar to `varSelRF` but applied to random forest with regression trees.  With this approach we fit a full random forest model that includes all variables and a large number of trees.  We then rank the variables using the increase in mean square error, which has been shown to be a less biased metric of importance than the mean decrease in the gini coeffecient [@strobl2007bias].  Using this ranking, we then iterate through the variables and create a random forests with the top two variables and record mean square error and adjusted R\textsuperscript{2} of the resultant random forest. We then repeat this process by adding the next most important variable in order of importance.  With this information we identify the top variables and the point at which adding variables does not improve the fit of the overall model.  These variables are selected and used as the "reduced model."  With this method, a minimum set of variables that maximizes model accuracy is provided.  This allows us to start with a full suite of predictor variables from which to select a minimum, easier to interpret set of variables.  
 
+<!---it is unclear to me how you are predictiong the "continuous variable".  Are you using the random forest similar to the CART process with the tree leading to a regression equation? If so, what are the predictor variables for the regression equations?  TN & TP? This is the section that needs more information. --->
```

Comment:
The predictor variables are the "All Variables" or "GIS Only"  and the tree is built as a regression tree as opposed to classification tree with each split being determined by a regression equation (my regression tree basics are a bit shaky).  Will add some stuff here to address this.